core-bundle/src/Resources/contao/models/PageModel.php

```diff
@@ -111,6 +111,10 @@
  * @property string|boolean $hasMooTools
  * @property string         $template
  * @property string         $templateGroup
+ * @property string         $newsArchives
+ * @property string         $feedFormat
+ * @property string         $feedSource
+ * @property integer        $maxFeedItems
```

Comment:
not sure we're adding properties from extensions (`news-bundle`) in files from the core bundle @contao/developers ?

Reply:
Oh, yes you are probably right. Did not think about this. Can remove it, if required?

Reply:
We don't add anything to the other models either. Removed these in 0bd0c6361